tests/unit/test_session.py

```diff
@@ -16,6 +17,7 @@ async def test_session_get_client(client_name, client_class):
         assert isinstance(client, client_class)
 
 
+@pytest.mark.asyncio
```

Comment:
I lost these along the way. It would have helped if pytest failed instead of skipping. Maybe we can configure it to do so. I don't immediately see how, it seems particular to async functions.

Reply:
agreed. will keep an eye out for a way to do this.